x/params/keeper_test.go

```diff
@@ -33,248 +44,92 @@ func TestKeeper(t *testing.T) {
 		{"key3", 182},
 		{"key4", 17582},
 		{"key5", 2768554},
+		{"store1/key1", 1157279},
+		{"store1/key2", 9058701},
 	}
 
 	skey := sdk.NewKVStoreKey("test")
-	ctx := defaultContext(skey)
-	setter := NewKeeper(codec.New(), skey).Setter()
+	tkey := sdk.NewTransientStoreKey("transient_test")
+	ctx := defaultContext(skey, tkey)
+	store := NewKeeper(codec.New(), skey, tkey).Substore("test")
 
 	for _, kv := range kvs {
-		err := setter.Set(ctx, kv.key, kv.param)
-		assert.Nil(t, err)
+		require.NotPanics(t, func() { store.Set(ctx, kv.key, kv.param) })
 	}
 
 	for _, kv := range kvs {
 		var param int64
-		err := setter.Get(ctx, kv.key, &param)
-		assert.Nil(t, err)
-		assert.Equal(t, kv.param, param)
+		require.NotPanics(t, func() { store.Get(ctx, kv.key, &param) })
+		require.Equal(t, kv.param, param)
 	}
 
 	cdc := codec.New()
 	for _, kv := range kvs {
 		var param int64
-		bz := setter.GetRaw(ctx, kv.key)
-		err := cdc.UnmarshalBinary(bz, &param)
-		assert.Nil(t, err)
-		assert.Equal(t, kv.param, param)
+		bz := store.GetRaw(ctx, kv.key)
+		err := cdc.UnmarshalJSON(bz, &param)
+		require.Nil(t, err)
+		require.Equal(t, kv.param, param)
 	}
 
 	for _, kv := range kvs {
 		var param bool
-		err := setter.Get(ctx, kv.key, &param)
-		assert.NotNil(t, err)
+		require.Panics(t, func() { store.Get(ctx, kv.key, &param) })
 	}
 
 	for _, kv := range kvs {
-		err := setter.Set(ctx, kv.key, true)
-		assert.NotNil(t, err)
+		require.Panics(t, func() { store.Set(ctx, kv.key, true) })
 	}
 }
 
-func TestGetter(t *testing.T) {
+func TestGet(t *testing.T) {
 	key := sdk.NewKVStoreKey("test")
-	ctx := defaultContext(key)
-	keeper := NewKeeper(codec.New(), key)
+	tkey := sdk.NewTransientStoreKey("transient_test")
+	ctx := defaultContext(key, tkey)
+	keeper := NewKeeper(createTestCodec(), key, tkey)
 
-	g := keeper.Getter()
-	s := keeper.Setter()
+	store := keeper.Substore("test")
 
 	kvs := []struct {
 		key   string
 		param interface{}
+		zero  interface{}
+		ptr   interface{}
 	}{
-		{"string", "test"},
-		{"bool", true},
-		{"int16", int16(1)},
-		{"int32", int32(1)},
-		{"int64", int64(1)},
-		{"uint16", uint16(1)},
-		{"uint32", uint32(1)},
-		{"uint64", uint64(1)},
-		{"int", sdk.NewInt(1)},
-		{"uint", sdk.NewUint(1)},
-		{"rat", sdk.NewDec(1)},
+		{"string", "test", "", new(string)},
+		{"bool", true, false, new(bool)},
+		{"int16", int16(1), int16(0), new(int16)},
+		{"int32", int32(1), int32(0), new(int32)},
+		{"int64", int64(1), int64(0), new(int64)},
+		{"uint16", uint16(1), uint16(0), new(uint16)},
+		{"uint32", uint32(1), uint32(0), new(uint32)},
+		{"uint64", uint64(1), uint64(0), new(uint64)},
+		/*
+			{NewKey("int"), sdk.NewInt(1), *new(sdk.Int), new(sdk.Int)},
```

Comment:
Why are these commented? Can we update them for the new store - or delete otherwise?